test/stdlib/String_test.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: false
```

Comment:
Lots of tested methods mutate the string, so it's simpler to globally use mutable strings in this file.